content/components/dialog.mdx

```diff
@@ -0,0 +1,420 @@
+
+If the content area has overflow scrolling, a divider will be shown between the footer and content area. Otherwise, showing a divider is optional.
+
+<img
+  src="https://user-images.githubusercontent.com/293280/175850630-9f3df284-c69b-4d16-9e57-ea04a0202d04.png"
+  width="960"
+  alt=""
+/>
+<Caption>Use buttons in a footer to let the user complete a task.</Caption>
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/293280/175850639-a77d537e-2807-47d2-b456-a298d5f44918.png"
+      alt=""
+    />
+    <Caption>Footer with small buttons aligned to the right.</Caption>
+  </div>
+  <div>
+    <img
+      src="https://user-images.githubusercontent.com/293280/175850637-3b193552-b35f-4b07-b2d6-4ca546c8131a.png"
+      alt=""
+    />
+    <Caption>Footer with a secondary button aligned to the left.</Caption>
+  </div>
+</Box>
+
+### Backdrop
+
+Dialog displays with a **backdrop**, which dims the rest of the page. The backdrop is visible when the Dialog is centered on the page, or attached to a side as a sheet.
+
+By default, clicking on the **backdrop** will dismiss the Dialog. However, if the Dialog contains a form that can have unsaved changes, the **backdrop** won't dismiss the Dialog, regardless of the state of the form.
+
+### Spacing
+
+The edges of Dialog are free of text and heavy visuals to visually distinguish the Dialog as an elevated surface, alongside its shadow. Content within the body and header should remain within the 16px safe area. However, inner components that have transparent or discrete backgrounds may bleed into the padding for visual alignment.
+
+<img
+  alt=""
+  src="https://user-images.githubusercontent.com/293280/174925634-603a27cf-2281-4a3a-9078-d1b8138e0d09.png"
+  width="960"
+/>
+<Caption>
+  1. A search field in the Subheader region appears closer to the edges. While the search icon remains within the safe
+  area.
+  <br />
+  2. Close button placed `8px` away from the edges. Note that the "×" glyph remains within the safe area.
+  <br />
+  3. Action list items in the Body region.
+  <br />
+  4. Footer buttons always appear inside the safe area. <br />
+</Caption>
+
+## Size
+
+Dialog offers a range of sizes from `small` to `xlarge` with a default of `medium`. Each size provides a `max-height` and will automatically shrink to its body content. Dialogs will never extend beyond the viewport, and the body content must fit a narrow viewport with a minimum width of `320px`.
+
+### Small
+
+<img
+  width="1292"
+  alt=""
+  src="https://user-images.githubusercontent.com/18661030/187482408-97bcb753-62a9-4b7c-b02b-b94ba369203e.png"
+/>
+<Caption>Small Dialogs have a min width of 320px, and a max height of 256px.</Caption>
```

Comment:
> have a min width of 320px

I'm seeing these descriptions across the different dialog sizes, but doesn't the "min width" makes it sound like the dialog won't get smaller than that? All dialogs will shrink on smaller viewports.

Reply:
In the main description I mention: 

> Dialogs will never extend beyond the viewport, and the body content must fit a narrow viewport with a minimum width of 320px.

Is it not clear that the dialog will always retract to fit inside the viewport? My goal is that a designer reading this can get a sense of sizing, while also understanding that the size is variable depending on the available screen space.